setup/baseConfig.yaml

```diff
@@ -35,17 +35,11 @@ libfuse:
   # Streaming configuration
 stream:
   # If block-size-mb, blocks-per-file or cache-size-mb are 0, the stream component will not cache blocks. 
-  block-size-mb: <size of each block to be cached in memory while streaming (in MB). Default - 0 MB>
-  blocks-per-file: <maximum number of blocks, per file, that can be cached at any given time. Default - 0>
-  cache-size-mb: <total amount of memory that streaming can consume on the system. Default - 0 MB>
-  policy: lfu|lru|arc <cache eviction policy for cached blocks. Default - arc>
-  cache-on-file-handle: true|false <cache block per file handle instead of file-name. (better performance but increases memory consumption)>
-  disk-persistence: <persist blocks on disk when evicted from memory>
-  disk-cache-path: <path to perist evicted blocks>
-  disk-size-mb: <maximum disk space to be consumed by disk-persistence>
-  disk-timeout-sec: <number of seconds a persisted block is considered valid>
+  block-size-mb: <for read only mode - size of each block to be cached in memory while streaming (in MB). Default - 0 MB>
+  handle-limit: <total amount of memory that streaming can consume on the system. Default - 0 MB>
+  handle-buffer-size-mb: <capacity of cached data for per file handle. Default - 0>
```

Comment:
handle-buffer-size-mb is not required here. We are reading only total cache capacity and number of handles

Reply:
I thought we agreed in the meeting that its more user friendly to have it in the context of handle since we already have `handle-limit` its easier to understand handle buffer size.

Reply:
Yes, we agreed to get handle limit and total memory and in code as well you divide the total memory by handle to get per handle buffer size. Then why we are reading this "handle-buffer-size-mb" from config. Are we like two modes where either user can give total memory or per handle buffer size?